public/js/controllers.js

```diff
@@ -30,71 +30,64 @@ projectMigrationApp.controller('projectMigrationController', ['Projects', 'Migra
   var migratingUrl = $rootScope.urls.migratingUrl;
 
   Migrations.getMigrations(migratingUrl).then(function(result) {
-    $scope.migratingProjects = _.sortBy(result.data.entity, 'site_id');
 
-    $rootScope.status.migrations = moment().format('h:mm:ss');
-    $log.info(moment().format('h:mm:ss') + ' - migrating projects loaded');
-    $log.info(' - - - - GET /migrating');
-    if (result.data.entity.length) {
-      $log.warn('page load got one or more current migrations - will have to poll it every ' + $rootScope.pollInterval/1000 + ' seconds');
-      
-      PollingService.startPolling('migrationsOnPageLoad', migratingUrl, $rootScope.pollInterval, function(result) {
-        $scope.migratingProjects = _.sortBy(transformMigrations(result).data.entity, 'site_id');
+    if (result.status ===200) {
```

Comment:
Is it possible to get a null pointer exception here if result is not returned (timeout)?